source/assets/js/modules/profile/controllers/profile.controller.js

```diff
@@ -29,6 +29,25 @@ angular.module('locApp.modules.profile.controllers')
             $scope.languages = response;
           });
 
+      /**
+       * @ngdoc function
+       * @name checkURL
+       * @description
+       * Check if URI resolves
+       */
+      $scope.checkURL = function() {
```

Comment:
I'm curious what this is actually doing -- is it performing the GET to figure this out?  if so, wouldn't HEAD be a better choice?